### ISSUE TITLE
Make lastFlushID and lastCompactIT consistency checks less brittle

### ISSUE THREAD

**Issue by Manno15** [apache/accumulo#2509]
**Is your feature request related to a problem? Please describe.**
Based on the comment from PR #2397: [here](https://github.com/apache/accumulo/pull/2397#discussion_r783204419)

```
This -1 practice is extremely brittle. 
A good follow-on PR would be to do something else 
(constant vs -1, OptionalLong, Long+null, ??) to make it less brittle.
```

**Describe the solution you'd like**
The ideal solution would be to use an alternative to '-1' so the consistency check becomes less brittle.

**Comment by foster33** [comment]
Is this issue still relevant? I think this was already addressed in #2531. That PR seems to remove and replace the brittle `.orElse(-1)` lines in the consistency checks.

**Comment by milleruntime** [comment]
I think this has been fixed in #2531
